client/starwhale/utils/venv.py

```diff
@@ -189,7 +190,6 @@ def venv_install_req(
         return
 
     venvdir = str(venvdir)
-    req = str(req)
```

Comment:
we need to convert req to str, for example: Path type.  https://github.com/star-whale/starwhale/blob/main/client/starwhale/core/runtime/model.py#L346

venv_install_req is a basic function for the most dependencies installation.